### ISSUE TITLE
Can you please help me out in making the icons on gBar smaller a little

### ISSUE THREAD

**Issue by Subhashis2007** [scorpion-26/gBar#64]
I want the progress bars and rounded progress indicators to be a little bit small, I know a little bit html and css, but couldn't figure it out, how by modifying the config,

**Comment by scorpion-26** [comment]
What exactly do you mean with progress indicators? If you mean the sensors (The circular indicators for CPU, RAM, ... + the two network arrows), then I just added config options for them. Just update to the latest commit and add/modify ```SensorSize``` and ```NetworkIconSize``` to your needs (Example can be found in ```data/config```).  If you mean the audio sliders, you can make them thinner by decreasing the ```min-height``` property in both ```trough``` and ```highlight``` in the css.

**Comment by Subhashis2007** [comment]
yeah I handled the audio bar already, was talking about cpu ram, thanks a lot, bro
